modules/govuk/manifests/apps/performanceplatform_big_screen_view.pp

```diff
@@ -29,5 +34,18 @@
       ssl_only        => true,
       app_port        => 3058,
     }
+    # Since this is not a full app, we need to ensure env dir exists.
+    file { "/etc/govuk/${app_name}/env.d":
+      ensure  => $ensure_directory,
+      purge   => true,
+      recurse => true,
+      force   => true,
+      notify  => Govuk::App::Service[$title],
```

Comment:
I think this `notify` line needs to go - because the app is just static HTML there is no service to notify.